### PR TITLE
chore: Update vaadin commercial license (V22.1) [skip ci]

### PR DESCRIPTION
**Pull request by ZheSun88**

### CODE REVIEW

**Comment by ZheSun88** [comment]
this will need flow 9.1-snapshot to run.

File: vaadin-board-flow-parent/vaadin-board-flow/src/main/java/com/vaadin/flow/component/board/Board.java
Comment (alvarezguille) [inline vaadin-board-flow-parent/vaadin-board-flow/src/main/java/com/vaadin/flow/component/board/Board.java:1]: Some files like `Board.java` have double license header mentioning both CVDL and new commercial license and service terms

Comment (ZheSun88) [reply vaadin-board-flow-parent/vaadin-board-flow/src/main/java/com/vaadin/flow/component/board/Board.java:1]: rechecked all the files and updated.

**Review by alvarezguille — CHANGES_REQUESTED** [review]
* `Apache License` is still mentioned in some `README.md` files and 
* Some components have the new license headers but still have apache headers like `LoginForm` or `AppLayout`
* `CVDLv4` is mentioned in asciidoc, the whole ` vaadin-charts-flow-parent/documentation` directory can be removed as done in #4259

**Review by alvarezguille — APPROVED** [review]

**Pull request merged**